features/networking/sriov.feature

```diff
@@ -66,6 +69,806 @@ Feature: Sriov related scenarios
+    And the output should contain: 
+      | intel-netdevice       |
+      | syncStatus: Succeeded |
+      | vfID: 4               |
+    """
+    Given I switch to the first user
+    And I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+    Given I obtain test data file "networking/sriov/sriovnetwork/intelnetdevice.yaml"
+    Given I create sriov resource with following:
+      | cr_yaml       | intelnetdevice.yaml   |
+      | cr_name       | intel-netdevice-rhcos |
+      | resource_type | sriovnetwork          |
+      | project       | <%= cb.usr_project%>  |
+    Then the step should succeed
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource  | net-attach-def        |
+      | namespace | <%= cb.usr_project%>  |
+    Then the step should succeed
+    And the output should contain "intel-netdevice-rhcos"
+    """
+    And I use the "<%= cb.usr_project%>" project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/macvlan-conf-without-master.yaml"
+    When I run the :create admin command with:
+      | f         | macvlan-conf-without-master.yaml |
+      | namespace | <%= cb.usr_project %>            |
+    Then the step should succeed
+    Given I obtain test data file "networking/sriov/pod/sriov-macvlan.yaml"
+    When I run the :create client command with:
+      | f | sriov-macvlan.yaml |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=sriov-macvlan |
+    When I execute on the pod:
+      | /usr/sbin/ip | -d | link |
+    Then the output should contain:
+      | net1 |
+      | net2 |
+    When I execute on the pod:
+      | bash | -c | /usr/sbin/ip addr show net1 |
+    Then the output should contain "10.56.217"
+    When I execute on the pod:
+      | bash | -c | /usr/sbin/ip addr show net2 |
+    Then the output should contain "192.168.1"  
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-24713
+  @destructive
+  @admin  
+  Scenario: NAD can be also updated when networknamespace is change
+    Given the sriov operator is running well
+    Given I switch to the first user
+    And I have a project
+    And evaluation of `project.name` is stored in the :usr_project1 clipboard
+    Given I obtain test data file "networking/sriov/sriovnetwork/intelnetdevice.yaml"
+    Given I create sriov resource with following:
+      | cr_yaml       | intelnetdevice.yaml   |
+      | cr_name       | intel-netdevice-rhcos |
+      | resource_type | sriovnetwork          |
+      | project       | <%= cb.usr_project1%> |
+    Then the step should succeed
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource  | net-attach-def        |
+      | namespace | <%= cb.usr_project1%> |
+    Then the step should succeed
+    And the output should contain "intel-netdevice-rhcos"
```

Comment:
This can be shorten as in https://github.com/openshift/verification-tests/pull/1380/files#r494000991

Reply:
updated